packages/docs/docs/concepts/progressive-loading/volumeProgressive.md

```diff
@@ -0,0 +1,134 @@
+
+- lossy images are original resolution/bit depth, but lossy encoded
+- thumbnail images are reduced resolution images
+- byte range images are a prefix of the full resolution, followed by
+  retrieving the remaining data. This only works for images like HTJ2K encoded
+  in resolution first ordering.
+
+# Creating Partial Resolution Images
+
+Static DICOMweb has been enhanced to add the ability to create partial resolution
+images, as well as to serve up byte range requests. Some example commands
+for a Ct dataset are below:
+
+```bash
+# Create HTJ2K as default and write HTJ2K lossy to .../lossy/
+mkdicomweb create -t jhc --recompress true --alternate jhc --alternate-name lossy d:\src\viewer-testdata\dcm\Juno
+# Create JLS and JLS thumbnail versions
+mkdicomweb create -t jhc --recompress true --alternate jls --alternate-name jls /src/viewer-testdata/dcm/Juno
+mkdicomweb create -t jhc --recompress true --alternate jls --alternate-name jlsThumbnail --alternate-thumbnail /src/viewer-testdata/dcm/Juno
+# Create HTJ2K lossless and thumbnail versions (this is not required in general
+# when the top item is already lossless)
+mkdicomweb create -t jhc --recompress true --alternate jhcLossless --alternate-name htj2k  /src/viewer-testdata/dcm/Juno
+mkdicomweb create -t jhc --recompress true --alternate jhc --alternate-name htj2kThumbnail --alternate-thumbnail /src/viewer-testdata/dcm/Juno
+```
+
+Any other tools creating multipart/related encapsulated data can be used, as
+can using accept headers or parameters for a standard DICOMweb server.
+
+Note the data path for these is, in general the normal DICOMweb path with
+`/frames/` replaced by some other name.
+
+# Configuration
+
+See the volumeProgressive example for full details.
+
+No special configuration is required for HTJ2K images streaming, but that isn't
+particularly effective. The byte range request is preferably, but requires
+additional server side support for byte ranges. Other types require custom
+configuration.
+
+## HTJ2K Byte Range Configuration
+
+```javascript
+cornerstoneDicomImageLoader.configure({
+  retrieveOptions: {
+    default: {
+      '3.2.840.10008.1.2.4.96': {
+        streamingDecode: true,
+      },
+      default: {},
+    },
+    multipleFinal: {
+      default: {
+        range: 1,
+      },
+    },
+    multipleFast: {
+      default: {
+        // Note it is the decode that is streaming, but a range request to
+        // get the data, not a streaming request.
+        streamingDecode: true,
+        range: 0,
+        initialBytes: 64000,
+        decodeLevel: 0,
+      },
+    },
+  },
+```
+
+# Performance
+
+The performance gains on this vary quite a bit depending on size of data
+and capabilities of the DICOMweb server components. In general, a 350% speed improvement
+to first volume render is seen because of the interleaved images filling
+nearby images. For straight JLS versus HTJ2K, that depends on the compression
```

Comment:
It is unusual that we are giving numbers here; let's avoid that. It might be faster or slower depending on the backend, as you say.